sdk/src/Core/Amazon.Runtime/Internal/Util/Logger.cs

```diff
@@ -60,14 +58,54 @@ private void ConfigsChanged(object sender, PropertyChangedEventArgs e)
         private void ConfigureLoggers()
         {
             LoggingOptions logging = AWSConfigs.LoggingConfig.LogTo;
+            bool found = true;
             foreach (InternalLogger il in loggers)
```

Comment:
Since the `loggers` collection is no longer being filled in during the constructor this `foreach` loop never going to be entered.